### PR TITLE
Add licenses to Wanderer and Coalition outfits

### PR DESCRIPTION
**Pull request by Disiuze**
Summary:
- Adds licenses to All Wanderer, Coalition and Heliarch outfits
- Adds licenses to Coalition and Heliarch ships
- Removes the Coalition shipyards and outfits event in favour of granting the player a license
- Give the player Coalition license if they don't already have it

With help from @Elyssaen

### CODE REVIEW

File: data/coalition missions.txt [hunk 1]
```diff
@@ -126,3 +126,14 @@ mission "Coalition: Contributor"
 			`	"For the sake of peace it is," says the Saryd. "The Heliarch military, well trained to use weapons only for defense and only for justice, they are. And to place the needs of the Coalition over the ambitions of their own native species, each one committed is. No such discipline do civilians have."`
 			label end
 			`	Apparently nothing is for sale here, but they assure you that many Coalition worlds will now grant you access to their shipyards and outfitters.`
+
+mission "Coalition License"
```
Comment (Elyssaen) [inline data/coalition missions.txt]: I'd put "Patch" in the name and add a comment above that this is for pre-0.9.7 compatibility.

**Comment by Bladewood** [comment]
I think this brings up a problem:
> He does mention that the other Wanderer worlds have agreed to sell you some of their technology, and that you will be allowed to "[view, admire, envy]" their ships, but not to purchase them.

The Summer Leaf, Deep River, and Earth Shaper are locked behind the Wanderer license. So, if the ability to purchase Wanderer outfits is behind that same license, then once we get the license we'll be able to purchase their ships as well, which is not allowed, as per the quote.

**Comment by Disiuze** [comment]
Actually, it's the other way around, You can only buy the outfits when you get access to the ships, as the license is normally unlocked later than the outfits.

**Comment by Bladewood** [comment]
Oh. I thought something was wrong with my argument.

Well, at least I found a problem. (that will hopefully be fixed.)

File: data/coalition outfits.txt [hunk 2]
```diff
@@ -10,6 +10,8 @@
 
 outfit "Small Collector Module"
 	category "Power"
+	licenses
+			Coalition
```
Comment (endless-sky) [inline data/coalition outfits.txt]: These should all be indented with just one additional tab, not two.

File: data/coalition missions.txt [hunk 3]
```diff
@@ -127,81 +127,14 @@ mission "Coalition: Contributor"
...
-		shipyard "Coalition Basics"
-		shipyard "Saryd"
-		outfitter "Coalition Advanced"
-	planet "Warm Slope"
-		outfitter "Coalition Basics"
+#For pre-0.9.7 compatibility
+mission "Coalition License Patch"
+	invisible
+	landing
+	name "Bug, you shouldn't see this"
```
Comment (endless-sky) [inline data/coalition missions.txt]: I wouldn't bother to include a name line.

File: data/wanderer outfits.txt [hunk 4]
```diff
@@ -10,6 +10,8 @@
 
 outfit "Sunbeam"
 	category "Guns"
+	licenses
+			"Wanderer Outfits"
```
Comment (endless-sky) [inline data/wanderer outfits.txt]: Same here, should be one tab indented, not two.

**Comment by endless-sky** [comment]
Thanks, I hadn't even gotten around to suggesting that someone should fix this, yet!

File: data/wanderers start.txt [hunk 5]
```diff
@@ -242,33 +242,10 @@ mission "Visit Wanderers Again"
 	
 	on accept
 		set "language: Wanderer"
+		set "license: Wanderer Outfits"
```
Comment (endless-sky) [inline data/wanderers start.txt]: This should probably be added to the new "Wanderer Hail Patch" mission, as well, so old saved games will have access to Wanderer tech.

File: data/coalition weapons.txt [hunk 6]
```diff
@@ -10,6 +10,8 @@
 
 outfit "Bombardment Cannon"
 	category "Guns"
+	licenses
+			Heliarch
```
Comment (endless-sky) [inline data/coalition weapons.txt]: These tabs still need fixing.

File: data/coalition outfits.txt [hunk 7]
```diff
@@ -30,6 +34,8 @@ outfit "Large Collector Module"
 
 outfit "Small Reactor Module"
 	category "Power"
+	licenses
+		Coalition
```
Comment (jafdy) [inline data/coalition outfits.txt]: If I remember correctly this is one of the Heliarch outfits. If it is then it should be behind a Heliarch license.

Comment (Amazinite) [inline data/coalition outfits.txt]: It is. Reactors and weapons were both Heliarchs only.

Comment (endless-sky) [inline data/coalition outfits.txt]: Yes, the reactors should be Heliarch items. Of course, they aren't sold anywhere, so it won't matter much what license they're under.

**Pull request merged**